src/track/cueinfo.h

```diff
@@ -21,6 +21,14 @@ enum class CueType {
                       // sound; not shown to user
 };
 
+enum class CueFlag {
```

Comment:
Good decision to use bit flags instead of a single bool. Even if only for type safety ;)

Reply:
Yes, that was the main reason (and readability on the caller's side). It's also easier to add more flags later on if we need to ;-)